lib/dynamo/dynamo/user.py

```diff
@@ -9,10 +10,21 @@ def get_user(user):
     return response.get('Item')
 
 
-def get_max_jobs_per_month(user) -> int:
+def get_priority(user) -> Optional[int]:
     user = get_user(user)
     if user:
-        max_jobs_per_month = int(user['max_jobs_per_month'])
+        priority = user.get('priority')
+    else:
+        priority = None
+    return priority
+
+
+def get_max_jobs_per_month(user) -> Optional[int]:
+    user = get_user(user)
+    if user:
+        max_jobs_per_month = user['max_jobs_per_month']
```

Comment:
catch the case where a user has a record, but that record has no `max_jobs_per_month` field
```suggestion
        max_jobs_per_month = user.get('max_jobs_per_month')
```

Reply:
Hrmm, it's actually more complicated than that, since we'd want to return the default MONTHLY_JOB_QUOTA_PER_USER in this case, rather than None